packages/joy-media/src/channels/EditChannel.tsx

```diff
@@ -54,18 +54,16 @@ const InnerForm = (props: MediaFormProps<OuterProps, FormValues>) => {
     resetForm
   } = props;
 
-  const { myAccountId, myMemberId } = useMyMembership();
+  const { myAccountId, myMemberId, memberIsCurator, memberIsContentLead } = useMyMembership();
+  const isChannelOwner = entity && !isAccountAChannelOwner(entity, myAccountId)
 
-  if (entity && !isAccountAChannelOwner(entity, myAccountId)) {
-    return <JoyError title={`Only owner can edit channel`} />
+  if (!isChannelOwner && !memberIsCurator && !memberIsContentLead) {
```

Comment:
```suggestion
  if (!(isChannelOwner || memberIsCurator || memberIsContentLead)) {
```

Becayse I think that misses some conditions, it is easier to think of it like this:

```
if (isChannelOwner || memberIsCurator || memberIsContentLead) {
  // Allow
} else {
  // Deny
}
```